src/natcap/invest/ui_server.py

```diff
@@ -69,6 +69,7 @@ def get_invest_getspec():
     set_locale(request.args.get('language', 'en'))
     target_model = request.get_json()
     target_module = MODEL_METADATA[target_model].pyname
+    importlib.reload(natcap.invest.spec_utils)
     model_module = importlib.reload(
```

Comment:
It's necessary because a module is only evaluated the first time it's `import`ed. Subsequent `import`s don't re-evaluate the module unless we specifically `reload` it. There's not really any way to recursively `reload` a module, and its dependencies, and their dependencies...

Reply:
Thank you!